README.md

```diff
@@ -599,6 +599,13 @@ Controls the iOS silent switch behavior
 
 Platforms: iOS
 
+#### limitMaxResolution
```

Comment:
This is very good feature ! I love it :)
but I have 2 questions,
- Is it possible to enable / disable it on runtime ? what I see from the code (not tested) It is handled only at startup. would be good to add this info in documentation please ?
- Can it be better to limit resolution from video display size instead of screen size ? maybe it wouls be better for the futur to rename it to "limitMaxResolutionToScreenSize" ?

Thank you !

Reply:
@freeboub I changed the prop name to "limitMaxResolutionToScreenSize" which makes a lot more sense 👍 
As for allowing this during runtime, I think that it would be a lot of effort to do this and the feature is mostly useful to enable during startup and never change it so I'm leaving that as is for now.

Reply:
for the runtime it was just an open question :)
Thank you for the renaming !